Library/Formula/iojs.rb

```diff
@@ -0,0 +1,134 @@
+# Note that x.even are stable releases, x.odd are devel releases
+class Iojs < Formula
+  homepage "https://iojs.org/"
+  url "https://iojs.org/dist/v1.0.1/iojs-v1.0.1.tar.gz"
+  sha256 "c26052dad5d9ccd8a7b9134806994ebf2d217a57d6efd5633e01c5cc1dcdedc5"
+
+  conflicts_with "node", :because => "io.js includes a symlink named node for compatibility."
+
+  bottle do
+    revision 1
+    sha1 "a98a1df66cfb0712b14489186c46f7087ba35bd7" => :yosemite
+    sha1 "0cd45412840a67d5d65e6bc3c0c3bcf8bc23153c" => :mavericks
+    sha1 "977332381c033626b991002c27e738c144ebbaac" => :mountain_lion
+  end
+
+  deprecated_option "enable-debug" => "with-debug"
+
+  option "with-debug", "Build with debugger hooks"
+  option "without-npm", "npm will not be installed"
+  option "without-completion", "npm bash completion will not be installed"
+
+  depends_on :python => :build
+
+  fails_with :llvm do
+    build 2326
+  end
+
+  resource "npm" do
+    url "https://registry.npmjs.org/npm/-/npm-2.1.18.tgz"
+    sha1 "e2af4c5f848fb023851cd2ec129005d33090bd57"
+  end
+
+  def install
+    args = %W{--prefix=#{prefix} --without-npm}
+    args << "--debug" if build.with? "debug"
+
+    # This should eventually be able to use the system icu4c, but right now
+    # it expects to find this dependency using pkgconfig.
+    if build.head?
```

Comment:
Thanks @smockle, that's a useful discussion. If there's an option to turn that support back on in the ` configure ` script, we could always add an optional dependency on Homebrew's ` icu4c ` and then feed that to ` iojs ` to enable that support? Presuming ` iojs ` have kept the Node system of allowing optional system deps instead of shipped ones.